.github/workflows/flutter-build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: flutter pub get
 
       - name: Verify formatting
-        run: flutter format --output=none --set-exit-if-changed --line-length=120 .
+        run: flutter format --output=none --set-exit-if-changed --line-length=120 $(find lib test -name "*.dart" -not -name "*.mocks.dart")
```

Comment:
If you are wondering: This was necessary because the mock files are not formatted correctly (with our bigger line length). So we need to exclude them from the formatting in our CI.